File: src/main/java/org/social/social_network/entity/Picture.java
package org.social.social_network.entity;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.data.annotation.Id;
import org.springframework.data.relational.core.mapping.Table;

import java.time.Instant;
import java.util.UUID;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
@Table(name = "pictures")
public class Picture {

    @Id
    private UUID id;

    private byte[] content;

    private Instant created;

    private UUID postId;
}

File: src/main/java/org/social/social_network/entity/Picture.kt
package org.social.social_network.entity

import lombok.Data
import org.springframework.data.annotation.Id
import org.springframework.data.relational.core.mapping.Table
import java.time.Instant
import java.util.UUID

@Data
@Table(name = "pictures")
class Picture {
    @Id
     val id:UUID ?= null;
     val content: ByteArray ?=null
     val created : Instant ?= null
     val postId: UUID ?=null

}